core/app.js

```diff
@@ -90,6 +90,7 @@ app.use(function *(next) {
       this.USE_LAYOUT = 'layout';
     }
   }
+  captionClass = "undefined";
```

Comment:
Rather than having a global variable, can you affix this to `this` instead? I'm concerned that with a global we might run into issues where this gets overridden by simultaneous requests.